### PR TITLE
Signals field (recordId)

### PR DESCRIPTION
**Pull request by jenningsanderson**
Moving the places update to this repo

### CODE REVIEW

File: schema/defs.yaml [hunk 1]
```diff
@@ -66,6 +66,7 @@ description: Common schema definitions shared by all themes
         that referenced property will be specified in the overture list of
         approved sources from the Overture Data Working Group that contains
         the relevant metadata for that dataset including license source organization.
+        The recordid refers to the specific record within the dataset that was used.
```
Comment (vcschapp) [inline schema/defs.yaml:69]: It's IMO preferable to move field-specific documentation into the field itself. A `description` property could be added into the `record[I]d` property.

**Review by vcschapp — CHANGES_REQUESTED** [review]

**Review by danbeeee — COMMENTED** [review]
address Vic's comments, assuming I'm doing this right

File: schema/defs.yaml [hunk 2]
```diff
@@ -74,6 +74,9 @@ description: Common schema definitions shared by all themes
           type: string
         dataset:
           type: string
+        recordId:
```
Comment (varapmsft) [inline schema/defs.yaml:77]: This approach would result in a lot of duplication. For example, in Admin dataset for M5 TOM is providing most of the data and one name, ESRI providing bunch of names. To populate the sources we will need to repeat the ESRI provider id dozens of times.

Another approach would be to create a separate property to list provider name to provider id mappings once.

**Review by TristanDiet-TomTom — APPROVED** [review]

File: schema/defs.yaml (hunk 2 above)
Comment (vcschapp) [reply schema/defs.yaml:77]: That's a good point Vara. I'd be willing to just ship what we have to see how it goes down, and then if (as is perhaps likely) we feel things are way over-duplicated, we could move to a system with more "indirection" like you're suggesting.

**Review by vcschapp — APPROVED** [review]
```
                  ~.
           Ya...___|__..aab     .   .
            Y88a  Y88o  Y88a   (     )
             Y88b  Y88b  Y88b   `.oo'
             :888  :888  :888  ( (`-'
    .---.    d88P  d88P  d88P   `.`.
   / .-._)  d8P'"""|"""'-Y8P      `.`.
  ( (`._) .-.  .-. |.-.  .-.  .-.   ) )
   \ `---( O )( O )( O )( O )( O )-' /
    `.    `-'  `-'  `-'  `-'  `-'  .' CJ
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Pull request merged**